plugins/fabrik_element/fileupload/fileupload.php

```diff
@@ -1066,8 +1062,8 @@ protected function _getAllowedExtension()
 
 		if ($allowedFiles != '')
 		{
-			// $$$ hugh - strip spaces, as folk often do ".foo, .bar"
-			preg_replace('#\s+#', '', $allowedFiles);
+			// $$$ hugh - strip spaces and leading ., as folk often do ".bmp, .jpg"
+			preg_replace('#(\s+|^)\.?#', '', trim($allowedFiles));
```

Comment:
Think this would need to be \s* rather than \s+ if you want to add detection of the dot to it?  Otherwise it won't pick up dotted extensions without a preceeding space?  Also, in general, rather than modifying a $$$ comment someone else has inserted, add your own.  Like "$$$ paul - modified RE to pick up dots as well as space" or whatever.